media/css/master.scss

```diff
@@ -233,6 +237,59 @@ h6,
   padding-top: 50px;
 }
 
+// 1.3 General buttons
```

Comment:
Can you make it 1.3 Objects, and then `.o-btn` please? The benefit of SCSS, is that you needn't to rewrite each time `.btn:selector` making for an easy addition of the `o-` prefix.

You can use the SCSS cascading with `&` for that:
```css
parent 
  // Parent styles
  &:selector,
  &:selector2,
  &:selector3 {
    // Selector styles
  }
  &.class {
    // Adjacent class styles
  }
}
```

Reply:
Example
![image](https://user-images.githubusercontent.com/26201406/87700064-6dda3a80-c79e-11ea-8a2a-15792d842722.png)